src/sidepanel/SidePanel.css

```
@@ -17,65 +32,113 @@ main {
-    color: #a394ff;
+    color: var(--primary-color);
     border: none;
     cursor: pointer;
-    font-size: 1rem;
+    font-size: 1.2rem;
+    margin-left: 0.5rem;
+    transition: color var(--transition-duration);
 
     &:hover {
-      color: #8273d9;
+      color: darken(var(--primary-color), 10%);
```

Biome (1.9.4)

[error] Unexpected unknown function: darken

Use a known function instead.
See MDN web docs for more details.

(lint/correctness/noUnknownFunction)